app/domain/authentication/authn_k8s/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,1135 @@
+# Conjur Kubernetes Authentication (`authn-k8s`) Troubleshooting Guide
+
+## Overview
+
+This guide presents some tips and guidelines for troubleshooting the
+functionality of
+[Conjur Kubernetes authentication (`authn-k8s`)](https://docs.conjur.org/Latest/en/Content/Operations/Services/k8s_auth.htm)
+on a [Conjur](https://docs.conjur.org/).
+
+### Troubleshooting Kubernetes Authentication on Conjur OSS
```

Comment:
I feel like this could be a `##` section, rather than a sub-section of the `## Overview`

Reply:
Indeed.